tutorials/using-cloud-vpn-with-checkpoint/index.md

```diff
@@ -0,0 +1,383 @@
+|Network|`to-cp`| The GCP network the route attaches to.|
+|Destination|`10.0.0.0/24`| IP range Destination IP address.|
+|Priority|`1000`|Route priority.|
+|Next hop|`Specify the VPN tunnel.`| |
+|Next hop VPN tunnel|`gcp-to-cp-vpn-tunnel-1`| The Tunnel created.|
+
+![alt_text](Image_3.PNG)
+
+**Note:** Add ingress firewall rules to allow inbound network traffic as per your security
+policy.
+
+## Configuring Check Point Secrurity Gateway
+
+To create an Interoperable Device for Cloud VPN on the Check Point SmartConsole:
+
+**Step 1**. Open SmartConsole > **New** > **More** > **Network Object** > **More** > **Interoperable Device**.
+
+**Step 2**. Configure the IP address associated with Cloud VPN peer (external IP).
+
+**Step 3**. Go to **General Properties** > **Topology** and manually add Google cloud IP addresses.
+
+![alt_text](Image_4.PNG)
+
+**Step 4**. Create a star community.
+
+1. Open SmartConsole > **Security Policies** > **Access Tools** > **VPN Communities**.
+2. Click **Star Community**. The New Star Community window opens.
+3. Enter an **Object Name** for the VPN Community.
+4. In the **Center Gateways** area, click the plus sign to add a Check Point Security Gateway object for the center of the community.
+5. In the **Satellite Gateways** area, click the **plus** sign to add the GCP gateway object.
+
+![alt_text](Image_5.PNG)
+
+**Step 5**. Configure these ciphers for IKEv1.
+
+Go to **Encryption** and change the Phase 1 and Phase 2 properties according what is specified in the Cipher configuration settings on page 3.
+
+**Note:** Make sure you select Perfect Forward Secrecy (Phase 2). This example refers to IKEv1. You can also use IKEv2 in this scenario.
+
+![alt_text](Image_6.PNG)
+
+**Step 6**. Go to the **Advanced** tab and modify the Renegotiation Time.
+
+* **IKE for Phase 1**: 610 minutes
+* **IKE for Phase 2**: 10,800 seconds
+
+![alt_text](Image_7.PNG)
+
+**Step 7**. Configure the Access Control Rule Base and Install policy.
+
+For more information, see the R80.10 Site To Site VPN Administration [Guide](http://dl3.checkpoint.com/paid/ea/ea41387591dcba2a8d551ba39084e9e6/CP_R80.10_SitetoSiteVPN_AdminGuide.pdf?HashKey=1515459944_c0affaeb9262c888e85d660e781d604d&xtn=.pdf).
+
+# Configuring Route Based IPsec VPN Tunnel
+
+Below is a sample environment to walk you through set up of route based VPN. Make sure
+to replace the IP addresses in the sample environment with your own IP addresses.
+
+**Google Cloud Platform**
+
+|Name | Value|                             
+-----|------                                  
+|GCP(external IP)|`35.195.227.26`|
+|VPC CIDR|`10.132.0.0/20`|
+|TUN-INSIDE GCP|`169.254.0.1`|
+|GCP-ASN|`65000`|
+                        
+**Check Point**
+
+|Name | Value|
+-----|------
+|Check Point Security Gateway(external IP)|`199.203.248.181`|
+|Addresses behind Check Point Security Gateway|`10.0.0.10/24`|
+|TUN-INSIDE- CP|`169.54.0.2`|
+|CP Security Gateway ASN|`65002`|
+
+## Configuring Cloud VPN
+
```

Comment:
*If* you want to tie in all the config below, add some intro text here e.g. (general preview of what's next): 

To configure Cloud VPN, you need to configure Cloud Router to exchange routes between your VPC and on-premise networks and then configure a VPN gateway and tunnel for the gateway-Check Point connection.